code/game/turfs/simulated/floor_glass.dm

```diff
@@ -5,10 +5,7 @@
 	// Oldspace for people who don't have parallax.
 	icon = 'icons/turf/space.dmi'
 	icon_state = "0"
-
-	plane = SPACE_BACKGROUND_PLANE
-	dynamic_lighting = 0
-	luminosity = 1
+	plane = TURF_PLANE
```

Comment:
Please confirm if players will now be covered in darkness while floating through space or if they're still visible as they were prior to this change

Reply:
This code is for the glass floor, not space. I think there was a reason it was on that plane before, though. I'm not sure if that reason is still relevant.